website/docs/guides/cli.md

```diff
@@ -82,23 +78,23 @@ Use "mcap [command] --help" for more information about a command.
 
 Convert a ROS 1 bag file to mcap:
 
-{/* cspell: disable */}
+{/_ cspell: disable _/}
```

Comment:
These are broken now; probably need to be removed or replaced with html comments. Since it's no longer mdx `{ }` will not work